src/pages/en/persona.md

```diff
@@ -0,0 +1,603 @@
+---
+title: Persona
+description: The main goal of creating these personas was to better understand our target audience. By developing a greater understanding of our users. By developing a greater understanding of our users, we would be able to build a responsive site architecture for all existing and future content hosted on the Digital Accessibility Toolkit’s website.
+---
+
+### The creation of our persona
+
+The main goal of creating these personas was to better understand our target audience. By developing a greater understanding of our users. By developing a greater understanding of our users, we would be able to build a responsive site architecture for all existing and future content hosted on the Digital Accessibility Toolkit’s website.
+
+To do this we must:
+
+- Define our audience,
+- Understand their specific goals and needs,
+- Discover the relationship between the site’s content and the audiences’ goals and needs.
+
+### Framework
+
+Our approach is modular: we combined two types of persona to create a template that better represents our target audience. We used the two following framework:
+
+#### Target user persona
+
+- Focuses on the goal and needs of the average user
+
+#### Accessibility persona
+
+- Identifies the frustrations people living with disabilities and/or limitation face when using inaccessible products.
+
+### Disability types and inclusivity
+
+In our personas, we included a variety of disabilities and a spectrum of limitations:
+• Permanent disabilities
+• Temporary limitations
+• Situational disabilities/limitations
+
+We believe it is important to account for a wide range of human experiences. This is why for each disability type we included different possibilities of exclusion.
+
+### Seven (7) Persona
+
+5 broad disability types and 2 combined disability types
+
+- [Brandon – Auditory](#Brandon)
+- [Alasie – Vision](#Alasie)
+- [Yenah – Dexterity](#Yenah)
+- [Ike – Mobility](#Ike)
+- [Damien – Cognitive](#Damien)
+- [Valeria – Cognitive and Auditory](#Valeria)
+- [Yosef – Mobility and Speech](#Yosef)
+
+<div class="container">
+
+### Brandon – Auditory {#Brandon}
+
+<div class="row">
+
+<div class="col-md-6">
+
+#### <i class="far fa-address-card" aria-hidden="true" ></i> About
+
+<dl>
+  <dt>Pronouns</dt>
+  <dd> He/Him</dd>
+  <dt>Age</dt>
+  <dd> 52</dd>
+  <dt>Job</dt>
+  <dd> User Experience (UX).</dd>
+  <dt>Commonly used sites</dt>
+  <dd> IAAP, CSPS.</dd>
+  <dt>Disability:</dt>
+  <dd>Deaf since birth.
+  <p>Example of Temporary or situational</p>
+  <ul>
+    <li>Temporary hearing loss due to an ear infection in the right ear.</li>
+    <li>Construction noise.</li>
+  </ul>
+  </dd>
+</dl>
+</div>
+</div>
+<div class="row">
+<div class="col-md-6">
+
+#### <i class="fas fa-signal" aria-hidden="true"></i> Skills (aptitude)
```

Comment:
Change all the `<I>` for `<span>` it's not that big of a deal but more semantic just using `<span>` even though font awesome uses `<I>`.